Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/corefinement.h

```diff
@@ -38,14 +39,94 @@ namespace Polygon_mesh_processing {
 namespace Corefinement
 {
 /** \ingroup PMP_corefinement_grp
- *  %Default new-face visitor model of `PMPCorefinementVisitor`.
+ *  %Default visitor model of `PMPCorefinementVisitor`.
  *  All of its functions have an empty body. This class can be used as a
  *  base class if only some of the functions of the concept require to be
  *  overridden.
  */
 template <class TriangleMesh>
 struct Default_visitor;
 
+#ifdef DOXYGEN_RUNNING
+/** \ingroup PMP_corefinement_grp
+ * A model of `PMPCorefinementVisitor` that allows to extract non-manifold
+ * outputs from a corefinement based Boolean Operations.
+ *
+ * \tparam TriangleMesh a model of `HalfedgeListGraph`, `FaceListGraph`, and `MutableFaceGraph`
```

Comment:
At the beginning I made meshes const but then the vpm type are no longer the same so it can be confusing for a using passing them to the coref function and the visitor